### DISCUSSION TITLE
Happy Holidays! 🎄❄️🎉

### DISCUSSION THREAD

**Discussion by nikoniko78** [nikoniko78/scripts-java-physion#44] [General]
What holiday do you celebrate? This is a special event for the winter holidays! Feel free to chat, share your stories, and have fun! Event closes on 1/5/24.

**Comment by git900000** [comment]
Sooooo.... What manga have you read lately?

**Reply by git900000** [reply]
Also, should I invite a few others from Physion Requests? It's a little barren here.

**Reply by nikoniko78** [reply]
Hi! 
I recently got surgery... I finished reading the instructional JavaScriptの本, so I can learn more advanced techniques.

**Reply by git900000** [reply]
Nice! Did you get the group invite-type thing in email? It's for me and Axolotlife's SportsYou team.

**Reply by git900000** [reply]
Oh, I see you did. Welcome!

:D

**Reply by nikoniko78** [reply]
Sorry I have been inactive lately; it is because I am not available a lot when the holidays are nearby. I am going to be completely inactive once my status changes, so watch out, k?

**Comment by nikoniko78** [comment]
This event has ended.

**Comment by git900000** [comment]
Ok, but what do you mean by "Completely inactive"? You're going to be inactive for a short period of time, or...?
________________________________

**Comment by nikoniko78** [comment]
I was on school break.

**Reply by git900000** [reply]
Oooh! Ok!
